tests/test_readme.py

```
@@ -0,0 +1,180 @@
+            r"##\s+.*License",            # License section
+        ]
+        
+        for section_pattern in required_sections:
+            assert re.search(section_pattern, content, re.IGNORECASE), \
+                f"Required section not found: {section_pattern}"
+
+
+class TestREADMEContent:
+    """Test README.md content quality and accuracy."""
+    
+    def test_project_description_exists(self):
```

Ruff (0.12.2)

Missing return type annotation for public function `test_project_description_exists`

Add return type annotation: `None`

(ANN201)